custom_components/vesync/light.py

```diff
@@ -226,7 +226,10 @@ def brightness(self):
     @property
     def is_on(self):
         """Return True if night light is on."""
-        return self.smarthumidifier.details["night_light_brightness"] > 0
+        if self.device.config_dict['module'] == 'VeSyncAirBypass':
```

Comment:
Some fans don't have the config_dict object, in particular the PUR131S.  I suppose line 53 would be false so this should be ok, but I'm not 100% sure...

Reply:
Oh i didn't notice that. I was going to check the device type with `isinstance`, but then I saw the module property in the dict and thought that it would be cleaner to use it instead. I can change it to that if you think that it would safer.